### ISSUE TITLE
How to fine-tune on the DUT-RGBD dataset?

### ISSUE THREAD

**Issue by Mhaiyang** [nnizhang/S2MA#1]
Hi, I am interested in your wonderful work but I do not know how to fine-tune the model on the DUT-RGBD dataset. Could you share the fine-tune training code? Thanks!

**Comment by nnizhang** [comment]
Thanks for your interest.

I have uploaded the fine-tune training code.
You could run python finetune_DUT_RGBD.py to fine-tune the model on the DUT-RGBD dataset.

Hope it will help you.

**Comment by Mhaiyang** [comment]
Thanks a lot!

The fine-tune process is actually training the model on the 800 images in the DUT-RGBD dataset for another 40,000 iterations by setting the initial learning rate to 0.001. Is that right?

Besides, some CVPR 2020 papers (e.g., SSF) use DUT-RGBD(800) + NJUD(1485) + NLPR(700) as the training set while some  work use NJUD(1400) + NLPR(650) as the training set and then fine tune on the DUT-RGBD dataset. I am confused as the number of NJUD or NLPR is inconsistent. Which training set should I use? Your feedback would be appreciated! 

Thanks!

**Comment by nnizhang** [comment]
Yes, you are right!

The inconsistency problem of NJUD and NLPR in some works do exist recently. I just follow the pioneering works[1], [2].

[1] Junwei Han, Hao Chen, Nian Liu, Chenggang Yan, and Xuelong Li. Cnns-based rgb-d saliency detection via cross-view
transfer and multiview fusion. IEEE Transactions on Cybernetics, 48(11):3171–3183, 2017
[2] Hao Chen and Youfu Li. Progressively complementarity-aware fusion network for rgb-d salient object detection. In
CVPR, pages 3051–3060, 2018.
